lib/grape/validations/params_scope.rb

```diff
@@ -201,7 +205,7 @@ def configure_declared_params
         end
       end
 
-      def validates(attrs, validations)
+      def validates(attrs, validations, fail_fast = false)
```

Comment:
I think the argument should be an options hash here, so we can make it extensible for the future. So `def validates(attrs, validations, opts = {})`.